tests/_utils-tests/model.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * @license Copyright (c) 2003-20'INSERT'6, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import { getData } from '/tests/engine/_utils/model.js';
+import Document from '/ckeditor5/engine/treemodel/document.js';
+import Element from '/ckeditor5/engine/treemodel/element.js';
+import Text from '/ckeditor5/engine/treemodel/text.js';
+import Range from '/ckeditor5/engine/treemodel/range.js';
+
+describe( 'model test utils', () => {
+	let document, root, selection;
+
+	beforeEach( () => {
+		document = new Document();
+		root = document.createRoot( 'main', '$root' );
+		selection = document.selection;
+
+		selection.removeAllRanges();
+	} );
+
+	describe( 'getData', () => {
+		it( 'writes elements and texts', () => {
+			root.appendChildren( [
+				new Element( 'a', null, [ 'atext' ] ),
```

Comment:
FYI: 'atext' does not have to be in an array., `new Element( 'a', null, 'atext' )` would be fine as well.